utils/verificators.py

```diff
@@ -0,0 +1,62 @@
+import requests
+from loguru import logger
+from assertpy import assert_that, soft_assertions
+from cerberus import validator, Validator
+from .logging import Logger
+
+Logger()
+
+def is_in(object1, object2):
+    logger.debug(f'Checking is "{object1}" in "{object2}"...')
+    try:
+        assert_that(object1).contains(object2)
+    except AssertionError:
```

Comment:
no need in try catch, the test should fail if assertion doesn't pass, no need in such logs on fail, it will be informative from a test